bx_py_utils/test_utils/log_utils.py

```diff
@@ -9,3 +9,23 @@ class RaiseLogUsage(logging.Handler):
 
     def handle(self, record):
         raise AssertionError(f'Missing log capture for: "{self.format(record)}"')
+
+
+class NoLogs:
+    """
+    Context manager to Suppress all logger outputs
+    """
+
+    def __init__(self, logger_name: str):
+        self.logger_name = logger_name
+
+    def __enter__(self):
+        self.logger = logging.getLogger(self.logger_name)
+        self.origin_handlers = self.logger.handlers.copy()
```

Comment:
This is a useless copy, right? `self.logger.handlers` can just be stored if we assign to it in the next line.

Reply:
right. updated.